### PR TITLE
Add more tests for BoykovKolmorogovAlgorithm

### PR DESCRIPTION
**Pull request by juliohm**
In an attempt to spot a possible bug in the Boykov-Kolmogorov algorithm (#43), I am adding more tests to it.

### CODE REVIEW

File: test/boykov_kolmogorov.jl [hunk 1]
```diff
@@ -1,4 +1,155 @@
...
+    for (I1, I2) in [[(4,4), (5,4)], [(5,4), (5,5)],
+                    [(5,5), (5,6)], [(5,6), (6,6)]]
+      u = LinearIndices(sz)[I1...]
+      v = LinearIndices(sz)[I2...]
+      C[u,v] = C[v,u] = 1
+    end
+
+    flow, _, labels = maximum_flow(G, s, t, C, algorithm=BoykovKolmogorovAlgorithm())
+
+    # because there is only one bridge, and because
```
Comment (juliohm) [inline test/boykov_kolmogorov.jl]:
```suggestion
    # because there is only one bridge,
```

**Comment by juliohm** [comment]
This PR is ready for review. Please let me know if you would like me to use the `request review` button on GitHub in future PRs.

**Comment by matbesancon** [comment]
all good thanks, could you close #43 accordingly?

**Comment by juliohm** [comment]
I am still investigating to see if the bug is present here or somewhere else downstream. I will close that issue as soon as I get to it.

**Pull request merged**